src/App/VisibilityBar.tsx

```diff
@@ -32,13 +33,49 @@ const VisibilityBar: FC<{ isSidePanelEnabled: boolean }> = ({
     const isLogVisible = useSelector(isLogVisibleSelector);
     const autoScroll = useSelector(autoScrollSelector);
 
+    useHotKey({
+        hotKey: 'shift+m',
+        title: 'Show/Hide menu',
+        isGlobal: true,
+        action: () => dispatch(toggleSidePanelVisible()),
+    });
+
+    useHotKey({
+        hotKey: 'shift+r',
+        title: 'Clear log',
+        isGlobal: true,
+        action: () => dispatch(clear()),
+    });
+
+    useHotKey({
+        hotKey: 'shift+o',
+        title: 'Open log file',
+        isGlobal: true,
+        action: () => logger.openLogFile(),
+    });
```

Comment:
I'm guessing you may have discussed with Ketil about what the shortcuts should be?
For me `shift + o` seems like a good idea for opening something, and it allows us to keep `ctrl + o` for opening external files in e.g. PPK.
Is the shift key reserved for global shortcuts?

Reply:
Did some smalltalk, but will get a review of the current hotkeys and change if needed. That would be the last commit before merge (hopefully...)